src/test/java/org/broadinstitute/dsm/GBFTest.java

```diff
@@ -225,4 +269,61 @@ public void inputConfirmationGBFKit() throws Exception {
             Assert.fail("No apiKey found");
         }
     }
+
+    @Test
+    public void orderGBFKit() throws Exception {
+        String externalOrderNumber = "T93CNUCZ8QHBSNRI1DJA";
+
+        String orderLookup = "select \n" +
+                "distinct req.ddp_participant_id\n" +
+                "FROM prod_dsm_db.ddp_kit_request req\n" +
+                "left join ddp_kit kit on\n" +
+                "(req.dsm_kit_request_id = kit.dsm_kit_request_id)\n" +
+                "where\n" +
+                "req.ddp_instance_id = 9\n" +
+                "and\n" +
+                "req.external_order_number = ?";
+
+        DDPInstance instance = DDPInstance.getDDPInstanceWithRole("testboston", DBConstants.HAS_KIT_REQUEST_ENDPOINTS);
+          ArrayList<KitRequest> kitsToOrder = new ArrayList<>();
+
+          EasyPostUtil easyPostUtil = new EasyPostUtil(null,"PwI372wBHmVyrdBqa2NoeA");
+        Map<String, Map<String, Object>> elasticMap = ElasticSearchUtil.getDDPParticipantsFromES(instance.getName(), instance.getParticipantIndexES());
+
+        HashMap<Integer, KitRequestSettings> kitRequestSettings = KitRequestSettings.getKitRequestSettings("9");
+
+        GBFRequestUtil shipper = new GBFRequestUtil();
+
+        TransactionWrapper.inTransaction(conn -> {
+            try {
+                try (PreparedStatement stmt = conn.prepareStatement(orderLookup)) {
+                    stmt.setString(1, externalOrderNumber);
+
+                    try (ResultSet rs = stmt.executeQuery()) {
+                        while (rs.next()) {
+                            String ddpParticipantId = rs.getString("ddp_participant_id");
+                            DDPParticipant participant = ElasticSearchUtil.getParticipantAsDDPParticipant(elasticMap, ddpParticipantId);
+
+                            kitsToOrder.add(new KitRequest(ddpParticipantId, participant.getShortId(), participant, externalOrderNumber));
+
+                            for (KitRequestSettings kitRequestSetting : kitRequestSettings.values()) {
+                                if ("gbf".equalsIgnoreCase(kitRequestSetting.getExternalShipper())) {
+                                    kitRequestSetting.setCarrierTo("3rd Day Air Residential");
+                                    kitRequestSetting.setServiceTo(kitRequestSetting.getCarrierTo());
+                                    System.out.println(kitRequestSetting.getCarrierTo());
+                                    shipper.orderKitRequests(kitsToOrder, easyPostUtil, kitRequestSetting, null);
+                                }
+                            }
+
+                        }
+                    }
+                }
+            } catch (Exception e) {
+                logger.error("Could not order kit " + externalOrderNumber, e);
+            }
+
+            return null;
```

Comment:
I don't see any assertions here--can you add some tests with assertions?

Reply:
This was made to be used to order kits through gbf, not as a real test. Since the new PR I don't think we will need it anymore...